### PR TITLE
fix(typo): changed all 'modifie' to 'modify'

### PR DESCRIPTION
**Pull request by TristanElain**
There was a typo error, all 'modify' words where typed 'modifie'

### CODE REVIEW

**Comment by TristanElain** [comment]
As I'm not the one doing the merge, I think I can't resolve those conflict.
If that's not the case please tell me how to do so.

**Comment by ripoul** [comment]
Add as remote my repo and pull the master of my repo in your's branch PR.

Le lun. 21 oct. 2019 à 21:46, TristanElain <notifications@github.com> a
écrit :

File: map/templates/map/nav-bar.html [hunk 1]
```diff
@@ -2,8 +2,8 @@
 <nav class="navbar navbar-light bg-light" style="margin-bottom: 5px;">
     <a href="/" class="navbar-brand">Georide Road Trip</a>
     <span>
-        <a class="btn btn-outline-success my-2 my-sm-0" href="/modifie" role="button">Modifie Account</a>
-        <a class="btn btn-outline-danger my-2 my-sm-0" href="/disconect" role="button">Sign Out</a>
+        <a class="btn btn-outline-success my-2 my-sm-0" href="/modify" role="button">Modify Account</a>
+        <a class="btn btn-outline-danger my-2 my-sm-0" href="/disconect" role="button">Disconect</a>
```
Comment (ripoul) [inline map/templates/map/nav-bar.html]: You change Sign out to disconnect

Comment (TristanElain) [reply map/templates/map/nav-bar.html]: My bad, I miss-checked my conflicts thinking there was only my updates.

**Pull request merged**